### ISSUE TITLE
Stereo camera issue on iOS

### ISSUE THREAD

**Issue by ZioCain** [googlevr/gvr-unity-sdk#742]
Hello everyone, I downloaded the latest SDK (1.70.00) and applied it to a new project, then added the old script & resources files from old project using an older version on GVR (i think it was 1.30)

Build for android: no issue at all
![vrtuosoandroid](https://user-images.githubusercontent.com/20202431/31287248-ecce4762-aac0-11e7-9036-795043cd4d5e.jpg)
Build for iOS (after several edit on the project through POD, according to gvr homepage)
![vrtuosoios](https://user-images.githubusercontent.com/20202431/31287257-f4960b9c-aac0-11e7-80e4-ce606b59c188.jpg)

The code is exactly the same, I'm not editing anything at all in the VRsettings or GVR. All I do is create the two objects you see on the android screen and download a JSON file from internet.

I tried to use different cardboard viewers QR code, but in the end it was always wrong on iOS (tested on multiple devices)

A thing that might be interesting:
while running the `pod update` command, I don't see GVRSDK (1.70.00) but instead I see `GVRSDK (1.80.00)`, dunno if that might mean something.

How do I fix the view on iOS?

Additional infos:
Mac OS X
XCode 9
Unity 5.6.1f

Tested on:
iPhone6 ios 11.0.2
iPhone7s PLUS ios 11.0.2
Android Honor8Pro EMUI5.1 (Android 7 equivalent, i guess)

**Comment by ZioCain** [comment]
@fredsa yeah, that's exactly what I do:
1. I set minimum iOS to 10.0
2. Build the project (tried both update and replace)
3. closed XCode, opened terminal, ran those commands
4. opened the workspace -> run on iphone -> you've seen the result
4. opened the project -> run on iphone -> same result

**Comment by fredsa** [comment]
@ZioCain What version of Unity are you using?

Can you try creating a new / empty Unity project, setting building Platform to iOS, import the latest Google VR SDK for Unity, open and build the GVRDemo scene, open workspace in Xcode and build it?

**Comment by ZioCain** [comment]
As stated before: I'm currently using Unity 5.6.1f
![screen shot 2017-10-11 at 10 31 29](https://user-images.githubusercontent.com/20202431/31429835-60034d8e-ae6f-11e7-887c-bda90a472cad.png)


Did that with Gvr 1.70 and it started only after the pod thing, still the the same issue
Tried then to import the new package (1.100) and:
- without doing the pod thing: it crashes just after the launch screen, shows double camera on totally black screen
- after doing the pod thing: 
![img_0005](https://user-images.githubusercontent.com/20202431/31429684-fd81bf42-ae6e-11e7-8dae-ab4278748f05.PNG)

**Comment by CMDann** [comment]
I am also experiencing an issue with this. The same app built in Unity and deployed to both Android and iOS is having an issue with warping on the stereoscopic viewer. We are rendering 360-degree images in our application. Here is the issue

Android:
![screenshot_20171013-174531](https://user-images.githubusercontent.com/4128000/31569917-9c84f16e-b044-11e7-9485-7242f7ab6f30.png)
![screenshot_20171013-174517](https://user-images.githubusercontent.com/4128000/31569918-a0e38e8c-b044-11e7-918e-13e5646661cb.png)

Same view on iOS:
![img_0023](https://user-images.githubusercontent.com/4128000/31569962-07543dce-b045-11e7-8759-db2457ec849a.jpg)


Notice that on iOS the right eye is quite skewed. I am building the application on high sierra using xCode 9. I am also setting the view to landscape manually:
![screen shot 2017-10-13 at 6 27 51 pm](https://user-images.githubusercontent.com/4128000/31569943-d6e5b35c-b044-11e7-9f8b-2747fa0d1ddf.png)

I am using Unity 2017 as my editor:
![screen shot 2017-10-13 at 6 28 30 pm](https://user-images.githubusercontent.com/4128000/31569945-de68cf74-b044-11e7-89d9-6780b2180d6c.png)

I have also tried the pod trick, redownloading GVR, however, this did not fix my issue.

**Comment by fredsa** [comment]
@CMDann thanks for the "update update".

The skewed images you were seeing could certainly be explained by an incorrect rotation somewhere in the project. If you can determine steps to reproduce this problem starting with an empty project, please feel free to file a new issue with the steps required and we'll try to get this sorted out.

Note, to rule out issues with the Xcode project settings, a good diagnostic step to try is to build a clean Xcode project from Unity and see whether that new project has the same issue.
* If the new project also has the issue, please attach a sample Unity project.
* If the new project doesn't have the issue, you'll have be able to compare the settings in the new "working" project and the existing "not working" project.
